assets/js/admin.js

```diff
@@ -1119,6 +1119,36 @@
       return str === null || str.match(/^ *$/) !== null;
     }
 
+        // Event listener for the 'Check Terms' button click
+        $('.auto-terms-content.st-delete-unuused-terms').find('#check-terms-btn').on('click', function(e) {
+          e.preventDefault();
+        
+          var numberRarely = $('#number-delete').val();
+        
+          $('#terms-feedback').text(st_admin_localize.checking_terms_message); 
```

Comment:
Change this here and other place to `$('.auto-terms-content.st-delete-unuused-terms #terms-feedback')`